website/views/shared/tasks/lists.jade

```diff
@@ -24,9 +24,14 @@ script(id='templates/habitrpg-tasks.html', type="text/ng-template")
 
           // Add New
           form.task-add(name='new{{list.type}}form', ng-hide='obj._locked', ng-submit='addTask(obj[list.type+"s"],list)')
-            input(type='text', ng-model='list.newTask', placeholder='{{list.placeHolder}}', required)
+            textarea(rows='6', ng-model='list.newTask', placeholder='{{list.placeHolderBulk}}', ng-if='list.bulk', required)
+            input(type='text', ng-model='list.newTask', placeholder='{{list.placeHolder}}', ng-if='!list.bulk', required)
             button(type='submit', ng-disabled='new{{list.type}}form.$invalid')
               span.glyphicon.glyphicon-plus
+            span.help-block
```

Comment:
Two things here:
1) Does this need its own `.help-block` class? Couldn't the whole thing be expressed like this:
``` jade
            small.btn-link.pull-right(ng-click='toggleBulk(list)')
              span.add-multiple(ng-if='!list.bulk') Add Multiple
              span.add-single(ng-if='list.bulk') Add Single
```

2) Let's make "Add Multiple" and "Add Single" translatable, so add strings to the locales files like you did for the export strings.